daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala

```diff
@@ -319,40 +319,39 @@ class CLIConf(arguments: Array[String]) extends scallop.ScallopConf(arguments)
   shortSubcommandsHelp()
 
   // Global Options
-  val debug = opt[Option[String]](argName = "file", descr = "enable debugging. Optionally, read initial debugger commands from [file] if provided.")(optionalValueConverter[String](a => a))
-  val trace = opt[Boolean](descr = "run the debugger with verbose trace output")
-  val verbose = tally(descr = "increment verbosity level, one level for each -v")
-  val version = opt[Boolean](descr = "Show version of this program")
+  val debug = opt[Option[String]](argName = "file", descr = "Enable the interactive debugger. Optionally, read initial debugger commands from [file] if provided.")(optionalValueConverter[String](a => a))
+  val trace = opt[Boolean](descr = "Run this program with verbose trace output")
+  val verbose = tally(descr = "Increment verbosity level, one level for each -v")
+  val version = opt[Boolean](descr = "Show Daffodil's version")
```

Comment:
I don't feel strongly about this, but looking at a handful of common linux tools, it seems the standard is to start option descriptions with a lower case character.